sdk/storage/azure-storage-blob-changefeed/mypy.ini

```diff
@@ -0,0 +1,14 @@
+[mypy]
+python_version = 3.6
+warn_return_any = True
+warn_unused_configs = True
+ignore_missing_imports = True
+
+# Per-module options:
+
+[mypy-azure.storage.blob._generated.*]
```

Comment:
Is this necessary?

Reply:
i believe yes when using the latest autorest.

Reply:
@annatisch I think you are right, we don't need this since there's no _generated folder in changefeed package.
while why do we need [mypy-azure.core.*] will mypy throw error in dependency package?